mlflow/promptflow/__init__.py

```diff
@@ -262,6 +269,10 @@ def save_model(
                     logged_model = mlflow.promptflow.log_model(
                         flow, artifact_path="promptflow_model", model_config=model_config
                     )
+        example_no_conversion: If ``False``, the input example will be converted to a Pandas
+            DataFrame format when saving. This is useful when the model expects a DataFrame
+            input and the input example could be passed directly to the model.
+            Defaults to ``False``.
```

Comment:
```suggestion
        example_no_conversion: {{ example_no_conversion }}
```

Should we use `{{ example_no_conversion }}` if all the flavors have the same description?

Reply:
this can be a follow-up